textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/nameentityextractor/NEExtractor.java

```diff
@@ -0,0 +1,70 @@
+package edu.uci.ics.textdb.dataflow.nameentityextractor;
+
+import edu.uci.ics.textdb.api.common.ITuple;
+import edu.uci.ics.textdb.api.dataflow.IOperator;
+import edu.uci.ics.textdb.common.exception.DataFlowException;
+
+
+/**
+ * Created by Sam on 16/4/21.
+ */
+public class NEExtractor implements IOperator {
+
+
+    private IOperator operator;
+
+
+    public NEExtractor(IOperator operator) {
+        this.operator = operator;
+    }
+
+
+    /**
+     * @about Opens name entities matcher.
```

Comment:
Let's be careful about the grammar in the comments.  Change "name entities" to "name entity".